esrally/telemetry.py

```diff
@@ -2225,3 +2226,60 @@ def record(self):
         }
 
         self.metrics_store.put_doc(doc, level=MetaInfoScope.cluster)
+
+
+class FieldDiskUsage(TelemetryDevice):
+    """
+    Measures the space taken by each field
+    """
+
+    internal = False
+    command = "field-disk-usage"
+    human_name = "Analyze field disk usage"
+    help = "Runs the analyze index disk usage API after benchmarking"
+
+    def __init__(self, telemetry_params, client, metrics_store):
+        """
+        :param telemetry_params: The configuration object for telemetry_params.
```

Comment:
Is there a way to inject a list of the indices being operated on? I wouldn't need this if I could get that list.

Reply:
That's ok.